pkg/controller/linstorsatelliteset/linstorsatelliteset_controller.go

```diff
@@ -1053,6 +1053,75 @@ func newSatelliteDaemonSet(satelliteSet *piraeusv1.LinstorSatelliteSet, satellit
 	ds = daemonsetWithMonitoringContainer(ds, satelliteSet, drbdReactorConfig)
 	ds = daemonSetWithSslConfiguration(ds, satelliteSet)
 	ds = daemonSetWithHttpsConfiguration(ds, satelliteSet)
+	ds = daemonSetWithDrbdHostPaths(ds, satelliteSet)
+	return ds
+}
+
+func daemonSetWithDrbdHostPaths(ds *apps.DaemonSet, set *piraeusv1.LinstorSatelliteSet) *apps.DaemonSet {
+	if set.Spec.MountDrbdResourceDirectoriesFromHost {
+		fileType := corev1.HostPathFile
+		ds.Spec.Template.Spec.Volumes = append(ds.Spec.Template.Spec.Volumes, []corev1.Volume{
+			{
+				Name: "etc-drbd-conf",
+				VolumeSource: corev1.VolumeSource{
+					HostPath: &corev1.HostPathVolumeSource{
+						Path: "/etc/drbd.conf",
+						Type: &fileType,
+					},
+				},
+			},
+			{
+				Name: "etc-drbd-d",
+				VolumeSource: corev1.VolumeSource{
+					HostPath: &corev1.HostPathVolumeSource{
+						Path: "/etc/drbd.d",
+						Type: &fileType,
```

Comment:
Should be `&kubeSpec.HostPathDirectoryType`. Actually, seeing this now, why not add the a `HostPathFileType` to `kubeSpec`, since we already have that there?